components/table/FilterPanel.tsx

```diff
@@ -1,15 +1,16 @@
 import { usePathname, useRouter, useSearchParams } from 'next/navigation';
-import { clusters, ministries } from '@/constants';
+import { clusters, ministries, providers } from '@/constants';
 import { useRef, useState } from 'react';
 
 export default function FilterPanel() {
   const [showInactive, setShowInactive] = useState(false);
   const { replace } = useRouter();
   const pathname = usePathname();
   const searchParams = useSearchParams()!;
-  const clusterRef = useRef<HTMLSelectElement>(null);
+  const clusterProviderRef = useRef<HTMLSelectElement>(null);
   const ministryRef = useRef<HTMLSelectElement>(null);
-
+  const currentClusterProvider = pathname.includes('private') ? 'Cluster' : 'Provider';
```

Comment:
As the `lowercase` string value is more used below, we could consider initially setting the value in `lowercase` and `capitalize` it when displaying it on the UI.
```js
function capitalizeFirstLetter(string) {
    return string.charAt(0).toUpperCase() + string.slice(1);
}
```

Reply:
changed, thanks